app/src/app/session/email-verification.dialog.html

```diff
@@ -0,0 +1,11 @@
+  <div [ngSwitch]="emailSent">
+    <div *ngSwitchCase="true">
```

Comment:
Having a div inside a div here make the DOM complicated. You could instead use ng-container which does not introduce an additional element in the DOM.